src/NuGetPackageVerifier/build/NuGetPackageVerifier.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="VerifyPackages"
+          AfterTargets="Pack"
```

Comment:
Going by @Eilon \ @davidfowl's comments in https://github.com/aspnet/specs/pull/49, I think it might be fine to unblock this. 

That said, instead of having executables that get run, would it make sense to just these be MSBuild tasks? It would make things like logging and reporting errors much nicer (TeamCity has first class support for desktop msbuild loggers, we might need to port these for msbuild core). Additionally, it would also decouple tools from the life cycle. We could have tool target files export msbuild tasks and have the importing project compose these. Having the lifecycle spread out between the build project + individual tools might be harder to manage and tedious to debug if they never get invoked.